### DISCUSSION TITLE
Can't display a long list of DateObject or Quantity objects

### DISCUSSION THREAD

**Discussion by Hachann** [WLJSTeam/wljs-notebook#277] [Frontend & UI]
Hello,

`RandomDate[n]` outputs a list of Dates (`DateObject` Head) up to n = 147. Starting at n = 148, there is no output.

 `Differences @ RandomDate[n]` outputs a list of dates differences (`Quantity` Head) up to n = 206. Starting at n = 207, there is no output.

**Comment by JerryI** [comment]
This fits better Issues. I moved it to there ;)

**Comment by JerryI** [comment]
In general this is correct. We have probably seen something like this

![Screenshot 2024-09-05 at 13 03 43](https://github.com/user-attachments/assets/4ee8d566-420b-4731-8616-9b4c2f78bf50)

This is natural limitations of big outputs. If we allow bigger, then our editor (CodeMirror 6) struggles to render all of it. In order not to affect UX, we limit the maximum output length (as characters).
